src/helpers/consul/index.js

```diff
@@ -1,4 +1,5 @@
+export * from './documentErrorMessageGenerator';
 export * from './filterHelper';
 export * from './homeData';
-export * from './imageErrorMessageHelper';
+export * from './imageErrorMessageGenerator';
```

Comment:
i am thinking of having them in one file `errorMessageGenerator`? thinking of the future, do we want to have a lot of different variants of such generators under `/helpers/consul/` or does they fit in one file?

Reply:
That's what I thought too. I will find the solution for this.